src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+public class Task {
+    private String task;
+    private boolean isMarked;
+    public String getTask() {
+        return task;
+    }
+    public void setTask(String task) {
+        this.task = task;
+    }
+    public boolean isMarked() {
+        return isMarked;
+    }
+    public void setMarked(boolean marked) {
+        isMarked = marked;
+    }
```

Comment:
Setter methods for boolean variables must be of the form:
`void setFound(boolean isFound);`

So, consider renaming the parameter from marked to isMarked to enhance clarity and to better reflect the boolean nature of the variable.

```suggestion
    public void setMarked(boolean isMarked) {
        this.isMarked = isMarked;
    }
```